sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/E2ETests/HttpClientInstrumentationTests.cs

```diff
@@ -85,13 +88,12 @@ public async Task HttpRequestsAreCapturedCorrectly(string? queryString, int expe
                     ? path
                     : path + queryString;
 
-            // TODO: THIS NEEDS TO BE INVETIGATED. DISTRO SHOULD BE DISABLING REDACTION.
             string expectedQueryString = queryString is null
                     ? string.Empty
 #if NET9_0_OR_GREATER //Starting with .NET 9, HttpClient library performs redaction by default
                     : "?*";
```

Comment:
There is a bug in the disto. We have a logic to remove redaction, but it is not being followed. This needs to be updated according to the specifications.

Reply:
cc: @TimothyMothra

Reply:
TODO: NEED WORK ITEMS TO GET THIS DONE BEFORE STABLE

Reply:
I created a bug to track this.